java/java.j2seplatform/src/org/netbeans/modules/java/j2seplatform/platformdefinition/DefaultPlatformImpl.java

```diff
@@ -42,8 +48,15 @@ public class DefaultPlatformImpl extends J2SEPlatformImpl {
 
     public static final String DEFAULT_PLATFORM_ANT_NAME = "default_platform";           //NOI18N
 
-    private ClassPath standardLibs;
+    @StaticResource
+    private static final String WARNING_ICON = "org/netbeans/modules/java/j2seplatform/resources/warning.png"; //NOI18N
 
+    @NbBundle.Messages({
+        "TITLE_InvalidDefaultJDK=Invalid JDK Home detected",
+        "DESC_InvalidDefaultJDK=The jdk.home property does not set a path to a valid JDK." +
+                               "Please update netbeans_jdkhome in your netbeans.conf" +
+                               "or start netbeans with --jdkhome /path/to/JDK set."
```

Comment:
how comes that `NotificationDisplayer` doesn't support html or `\n` line breaks? This creates a fairly long line.

Reply:
I assume you tested the message starting with `<html>`?

Reply:
it simply rendered with the html tags in plain text. but Disregard this, we probably don't need a notification.

Reply:
Yes, disregard for this. Although I thought there was code in there to special case html. Will take a look sometime.